k9mail-library/src/main/java/com/fsck/k9/mail/store/imap/Capabilities.java

```diff
@@ -3,7 +3,6 @@
 
 class Capabilities {
     public static final String IDLE = "IDLE";
-    public static final String ENABLE = "ENABLE";
```

Comment:
Actually I was removing this. Since support for QRESYNC implies support for ENABLE, a check for this capability should not be required.